### PR TITLE
Implemented conversions between JSON and Xcode files

### PR DESCRIPTION
**Pull request by Jag-Marcel**
### Description

This PR implements python scripts to convert between JSON files and Xcode's .xcstrings string catalogs and adds the current localizations from the iOS app to the existing JSONs.

### CODE REVIEW

File: .github/workflows/xcstrings_conversion.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          $SourceDiff = $diff | Where-Object { $_ -match '^Scribe-i18n/' -and $_ -match '.json$' }
+          $HasDiff = $SourceDiff.Length -gt 0
+          
+          Write-Host "::set-output name=json_changed::$HasDiff"
+  # Runs xcstrings conversion script
+  conditional_job:
+    runs-on: ubuntu-latest
+    needs:
+      - conditional_job_check_files
+    if: needs.conditional_job_check_files.outsputs.json_changed == 'True'
```
Comment (andrewtavis) [inline .github/workflows/xcstrings_conversion.yml:31]: Just checking, but this should be `outputs` - i.e. extra s in the middle?

File: Scribe-i18n/pt.json
Comment (andrewtavis) [inline Scribe-i18n/pt.json:1]: We do need `pt.json` for the Portuguese localization :)

File: .github/workflows/xcstrings_conversion.yml (hunk 1 above)
Comment (andrewtavis) [reply .github/workflows/xcstrings_conversion.yml]: Was showing me an error on my end :)

**Review by andrewtavis — APPROVED** [review]
Let's look at this a bit more on Saturday, @Jag-Marcel! :) I did fix the minor things that I talked about in the comments. Let's discuss what we want the key naming criteria to be as a group, but feel free to start preparing the localization process in iOS as you see fit :)

**Comment by andrewtavis** [comment]
Would be great if you'd present this, if you're comfortable 😊

**Comment by Jag-Marcel** [comment]
Yes, I can present this at the next sync

**Comment by Jag-Marcel** [comment]
@wkyoshida it looks like the changes here aren't automatically updated in the Scribe-iOS subtree, could you sync them when you have the time?

**Pull request merged**